src/containers/Tag/TagAdd/TagAdd.tsx

```diff
@@ -25,9 +31,44 @@ export const TagAdd: React.SFC<TagAddProps> = (props: TagAddProps) => {
 	const [parentId, setParentId] = useState(tag ? tag.parent_id : "");
 	const [formSubmitted, setFormSubmitted] = useState(false);
 
+	const [nameError, setNameError] = useState(false);
+	const [descriptionError, setDescriptionError] = useState(false);
+	const [languageIdError, setLanguageIdError] = useState(false);
+	const [parentIdError, setParentIdError] = useState(false);
+
 	const tagActions = useActions(TagActions);
 
+	const emptyMessage = "Empty entry."
+	const negativeMessage = "Cannot have negative entries."
+
+	const validateEntries = () => {
+		let noError = true;
+		if (name === "") {
+			noError = false;
+			setNameError(true);
+		}
+		if (description === "") {
```

Comment:
This is a common use case for determining the required fields. So maybe abstract this into generic / utility function.